client/verta/docs/tutorials/packaging/registry.rst

```diff
@@ -0,0 +1,65 @@
+Register Models and Versions in Model Registry
```

Comment:
Split. It's achieving two things. At the end of registering a model, add a link to adding a version.

Reply:
Splitted.